crates/pop-cli/src/commands/fork/mod.rs

```diff
@@ -0,0 +1,44 @@
+// SPDX-License-Identifier: GPL-3.0
+
+use clap::{Args, Subcommand};
+use std::fmt::{Display, Formatter, Result};
+use url::Url;
+
+#[cfg(feature = "chain")]
+pub(crate) mod chain;
+#[cfg(any(feature = "polkavm-contracts", feature = "wasm-contracts"))]
+pub(crate) mod contract;
+
+/// Arguments of the fork command.
+#[derive(Args)]
+#[command(args_conflicts_with_subcommands = true)]
+pub(crate) struct ForkArgs {
+	/// Entrypoint of the fork command.
+	#[command(subcommand)]
+	pub(crate) command: Command,
```

Comment:
could be an Option and if no subcommand is provided to fork default to `fork chain`